lib/src/st2110/st_tx_video_session.c

```diff
@@ -2360,7 +2397,7 @@ static int tv_mempool_init(struct mtl_main_impl* impl,
       info("%s(%d), use tx mono hdr mempool(%p) for port %d\n", __func__, idx,
            s->mbuf_mempool_hdr[i], i);
     } else {
-      n = mt_if_nb_tx_desc(impl, port) + s->ring_count + 512;
+      n = mt_if_nb_tx_desc(impl, port) + s->ring_count + 1024;
```

Comment:
Can we refine the hardcode 1024 to the ring size of rtcp buffer? And seems it should be set to 0 if rtcp is not enabled.